src/libs/convertToLTRForComposer/index.android.ts

```diff
@@ -1,8 +1,62 @@
+import CONST from '@src/CONST';
 import ConvertToLTRForComposer from './types';
 
+/**
+ * Android only - The composer can be converted to LTR if its content is the LTR character followed by an @ or space
```

Comment:
This comment should also explain _why_ it's OK to convert it in these cases and _why_ it shouldn't be converted. Consider adding "because..." and then fill out the rest.